stages/org.osbuild.tar

```diff
@@ -61,9 +61,10 @@ def main(inputs, output_dir, options):
     # Make a tarball of the tree.
     subprocess.run(
         tar_cmd,
-        stdout=subprocess.DEVNULL,
         check=True,
-        env=tar_env
+        env=tar_env,
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE
```

Comment:
What am I missing here?  We're capturing the output what not doing anything with it?

Reply:
For now only the test checks for return code 2 and "Wrote only … bytes" which the application (`org.osbuild.tar`) would also need to "detect" that `tar` didn't work properly due to disk full and osbuild would need to report "disk full" to the upper layers then.

Reply:
… at least those where a user needs to do something about it

Reply:
@achilleas-k the implementation was hiding the output before btw. 🤔
`stdout=subprocess.DEVNULL,`
but I only need stderr in this case, so the `DEVNULL` can stay

Reply:
@achilleas-k thanks for the reminder - I was using the wrong capture method in pytest…